src/main/java/core/basesyntax/service/strategy/OperationService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.TemporaryStorage;
+
+public interface OperationService {
+    void calculateByOperation(FruitTransaction fruitTransaction);
+
+    default boolean isOnBalanceSheet(FruitTransaction fruitTransaction) {
+        return TemporaryStorage.temporaryStorage.containsKey(fruitTransaction.getFruit());
```

Comment:
What if we have fruit with quantity = 0? Is it on balance or not? 
I'd recommend to remove  method `isOnBalanceSheet()` at all, and use use  `Map.getOrDefault()` instead of `Map.get()` where it needed. You can investigate what the difference between them.

Reply:
I don't quite understand this comment.